### ISSUE TITLE
Hide the amount of payment

### ISSUE THREAD

**Issue by Eloren1** [RTUITLab/ITLab-Reports-Front#37]
Добавить возможность скрывать сумму в **событиях** и **отчетах**, чтобы можно было открывать их где угодно.

<img src="https://user-images.githubusercontent.com/46666053/134042236-23b59511-ec2f-4480-b710-1ec5714831fe.png" width="40%">

<img src="https://user-images.githubusercontent.com/46666053/134042391-c7e3931c-ad33-4cbb-a53e-b7e634e6dc97.png" width="40%">

**Comment by CAPCHIK** [comment]
@Eloren1, то есть на всем сайте отключать глобально как-бы?

**Comment by Eloren1** [comment]
@CAPCHIK да, сделав параметр в настройках. При наведении на прямоугольник появляется сумма.

**Comment by CAPCHIK** [comment]
@Suput, идея выглядит классной, и на самом деле полезной. Добавил в draft ITLab, надо будет продумать как когда и кем это сделать.

**Comment by Suput** [comment]
> @CAPCHIK да, сделав параметр в настройках. При наведении на прямоугольник появляется сумма.

Сразу заинтересовал вопрос с мобилкой: а как там?
В голову пришло: при нажатии пальцем на прямоугольник, показывать сумму, нажал снова - скрыл сумму
